src/main/java/com/reckue/post/controller/CommentController.java

```diff
@@ -1,7 +1,7 @@
 package com.reckue.post.controller;
 
+import com.reckue.libs.exception.ReckueUnauthorizedException;
```

Comment:
Change the `ReckueException` to `BaseException`. 
Because the name of the service may change in the future.